sys-cluster/easybuild-easyblocks/easybuild-easyblocks-4.9.0.ebuild

```
@@ -0,0 +1,37 @@
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..11} )
```

```suggestion
PYTHON_COMPAT=( python3_{10..11} )
```

Old python versions should be dropped otherwise we'll get annoying complaints from portage.

---

(these changes should be made to the other files too, sorry, I wasn't clear before)

---

Nope, lazy me on the train just only hit the commit suggestion button.